k8s/baka/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.1
```

Comment:
NO WAY YOU AKSHUALLY DID IT 
![shock](https://user-images.githubusercontent.com/49134679/204399981-f6fa89fb-cc4d-4cc7-8ea5-bc91e220eeb3.png)